timeseries-streaming/timeseries-python-applications/MLPipeline/timeseries/pipeline_templates/timeseries_pipeline.py

```diff
@@ -54,6 +52,7 @@ def create_pipeline(
         ai_platform_training_args: Optional[Dict[Text, Text]] = None,
         ai_platform_serving_args: Optional[Dict[Text, Any]] = None,
         trainer_custom_config: Optional[Dict[Text, Any]] = None,
+        transformer_custom_config: Optional[Dict[Text, Any]] = None,
 ) -> pipeline.Pipeline:
     components = []
 
```

Comment:
ImportExampleGen is used below, so streaming beam data is already materialized in data_path, right?

no streaming from start of TFX just like [this](https://github.com/tensorflow/tfx/pull/2377)?

Reply:
Yes the data is 'continuously' materialised to the Data location. However there will be a separate PR raised for the sample to make use of a custom example gen next week. The custom example gen will be used to ensure train and test data have temporal separation.

Reply:
@rezarokni Has this custom example gen been implemented?

Reply:
@ConverJens Not yet. Now that this was merged can spend more time on it.